### PR TITLE
Adding third n/a option to 11B. Closes #325

### LINKED ISSUE #325: 11B is actually optional
We should make it optional and have a N/A

### PR DESCRIPTION
**Pull request by webinista**

### CODE REVIEW

File: app/views/web/questions.html.erb [hunk 1]
```diff
@@ -225,10 +219,16 @@
...
+      <%= render partial: 'form8_multi_radio', locals: {
               question: 'Have the requirements of 38 U.S.C. 7105A been followed for all contested claims?',
               question_num: '11B',
               input_name: '11B_HAVE_THE_REQUIREMENTS_OF_38_USC_BEEN_FOLLOWED',
+              option1_label: 'Yes',
+              option1_value: 'yes',
+              option2_label: 'No',
+              option2_value: 'no',
+              option3_label: '<abbr title="not applicable">N/A</abbr>',
+              option3_value: 'NOT_APPLICABLE',
```
Comment (alex) [inline app/views/web/questions.html.erb]: So, in the paper form there's two checkboxes, "Yes" and "No". What is the expected behavior if N/A is checked, and have we confirmed that this does the right thing with the resulting PDF?

Comment (paultag) [inline app/views/web/questions.html.erb]: When N/A is marked, it remains empty, which looks good to me!

**Comment by paultag** [comment]
Marking this as 'No' doesn't render in the PDF as no

**Comment by paultag** [comment]
Neither is yes, checking master now

**Comment by paultag** [comment]
Also apparently not working on master, either

**Comment by paultag** [comment]
Ignore me, we delete if `11A` is set. Re-testing

**Comment by paultag** [comment]
`11A` marked as `Yes` and `11B` marked as `N/A` marks it `No`. Is, uh, is that right? I actually have no idea.

**Comment by paultag** [comment]
(Yes, I'm fine with this behavior, on further thought), others?

**Comment by alex** [comment]
I think leaving it blank would be most correct?

**Comment by paultag** [comment]
@alex it's required if the `11A` is `Yes`

**Comment by alex** [comment]
Misread your comment, you shouldn't be able to enter `N/A` if `Yes` is selected in `11A`. This furthers my position that we really need to fix the underlying "hide fields" logic, it's too confusing otherwise.

**Comment by paultag** [comment]
@alex granted - filed a ticket as https://github.com/department-of-veterans-affairs/caseflow/issues/332 for the long-term fix

**Comment by paultag** [comment]
Gonna need a @KHarshawat call on this one :)

**Comment by plusjeff** [comment]
Agreed that we should fix the underlying hide functionality, so I'm glad there's a ticket to do that. Also, it might be relevant that N/A is forced by the controller layer if the user accidentally selects a value after 11A is 'No'.

https://github.com/department-of-veterans-affairs/caseflow/blob/issue%23325/app/controllers/web_controller.rb#L109

**Comment by KHarshawat** [comment]
Agree the long term fix is to use the show/hide logic. @plusjeff also works for right now but if we're implementing the logic to disable the question when 11a is No, we might as well hide the field altogether. 

So let's start by hiding this field if 11b is no and make sure we test it out thoroughly before pushing to Prod.

**Comment by webinista** [comment]
> So let's start by hiding this field if 11b is no and make sure we test it out thoroughly before pushing to Prod.

@paultag @plusjeff @alex: What is the property name of the answer for 11B? For 10A, it's `@kase.hearing_requested` but I can't tell what it is for 11B.

**Comment by plusjeff** [comment]
@webinista `11B_HAVE_THE_REQUIREMENTS_OF_38_USC_BEEN_FOLLOWED`

**Comment by alex** [comment]
@webinista there's no initial value, this is something they need to answer that's not in the database yet, and the only place it ends up going is the Form8 PDF.

**Comment by webinista** [comment]
Thank you @alex. That's the point I needed clarity about.

**Comment by alex** [comment]
:+1:

**Comment by webinista** [comment]
Some files are in here that shouldn't be. Going to use another PR.

**Pull request closed without merging**